src/utils/urlUtil.ts

```diff
@@ -242,5 +229,14 @@ export const urlUtil = {
    */
   removeTrailingSlashes(url: string): string {
     return url.replace(/\/+$/, '');
+  },
+
+  getUrlRelativePath(url: string): string {
+    if (url.indexOf('://') > 0 || url.indexOf('//') === 0) {
+      const parsedUrl = new URL(url);
+      const tenantUrl: string = `${parsedUrl.protocol}//${parsedUrl.hostname}`;
+      return url.substring(tenantUrl.length);
```

Comment:
Can't we use `parsedUrl.origin` to get the root URL?

Reply:
yes